Makefile

```diff
@@ -1,5 +1,9 @@
 # Include variables from ENV file
-include .env
+ENV =
+-include .env
```

Comment:
This makes it optional include the `.env` file, in case that the `.env` file is missing it won't raise the error.